sites/broadbandtechreport/server/templates/content/index.marko

```diff
@@ -83,6 +83,16 @@ $ const displayPublishedDate = ["event", "webinar", "contact"].includes(type) ?
                   <@link class="btn btn-lg btn-primary" />
                 </default-theme-content-download>
               </if>
+              <if(type === "webinar")>
```

Comment:
Can you confirm that all the other sites have this?

Reply:
For the EBM repo:

Has "webinar" and "product":
- CIM
- ILS
- LFW
- LED
- LW
- MAE
- VSD

Does not have "webinar" or "product":
- BOW
- DIQ
- EE
- SU (pretty sure this one doesn't have webcasts anyway)

Has "webinar", but not "product":
- DE
- FCN
- IAS
- OS
- PIA
- PM
- RDH
- UP
- WTO

Reply:
So, BOW has 2 webcasts, SU has 1 webcast, and DIQ/EE have 0 webcasts on their sites.